konig-core/src/main/java/io/konig/shacl/Shape.java

```diff
@@ -90,9 +90,18 @@
 	private URI rollUpBy;
 	private String mediaTypeBaseName;
 	private List<URI> inputShapeOf;
+	private List<Value> oneOf;
```

Comment:
Remove this property.  Values should be stored in the existing property given by

```
	private OrConstraint or;
```

You shoujld not have to modify the `Shape` class.